pages/blog/[slug].tsx

```diff
@@ -49,7 +49,9 @@ export default function SingleArticlePage(props: InferGetStaticPropsType<typeof
     }
   }, []);
 
-  const { slug, content, data } = props;
+  const { slug, data } = props;
+  const content = data.getPostsDocument.data.body;
```

Comment:
This allows the `content` variable to be live updated

Reply:
We had that before, I must have done something wrong while rebasing 🤔  Anyways, that makes perfect sense, thanks 😄